backend/app/api/endpoints/groups.py

```
@@ -938,6 +939,56 @@ def transfer_ownership_endpoint(
 # ============================================================================
 
 
+@router.get(
+    "/{group_name:path}/with-children", response_model=GroupWithChildrenResponse
+)
+def get_group_with_children_endpoint(
+    group_name: str = Path(
+        ..., description="Group name (may contain slashes for subgroups)"
+    ),
+    current_user: User = Depends(get_current_user),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)